File: 27522.py
import sys

rank_table = sorted(map(lambda x : x.strip().split(), sys.stdin.readlines()))
r, b = 0, 0
score = [10,8,6,5,4,3,2,1]

for i, team in enumerate(rank_table):
	if team[1] == 'B':  b += score[i]
	else: r += score[i]

print('Blue' if r < b else 'Red')

File: 27522.cpp
#include <bits/stdc++.h>
#define ALL(X) X.begin(), X.end()
using namespace std;

vector<pair<string, string>> rank_table(8);

void input() {
	for(auto& [time, team] : rank_table) {
		cin >> time >> team;
	}
	sort(ALL(rank_table));
}

void sol() {
	int score[] = {10,8,6,5,4,3,2,1};
	int red = 0, blue = 0;

	for(int i = 0; i < 8; i++) {
		auto& [_, team] = rank_table[i];

		if (team == "B") blue += score[i];
		else red += score[i];
	}
	cout << (red < blue ? "Blue" : "Red");
}

int main() {
	cin.tie(0)->sync_with_stdio(false);
	input();
	sol();
}